Documentation/Books/Manual/Architecture/StorageEngines.md

```diff
@@ -64,7 +64,29 @@ will avoid write conflicts but also inhibits concurrent writes.
 Currently, another restriction is due to the transaction handling in
 RocksDB. Transactions are limited in total size. If you have a statement
 modifying a lot of documents it is necessary to commit data inbetween. This will
-be done automatically for AQL by default.
+be done automatically for AQL by default. Transactions that get too big (in terms of
+number of operations involved or the total size of data modified by the transaction)
+will be committed automatically. Effectively this means that big user transactions
+are split into multiple smaller RocksDB transactions that are committed individually.
+The entire user transaction will not necessarily have ACID properties in this case.
+
+The threshold values for transaction sizes can be configured globally using the
+startup options
+
+* `--rocksdb.intermediate-commit-size`: if the size of all operations in a transaction 
```

Comment:
My opinion: I would just list the 3 options here, without explaining in details their meaning, and then add link to the page where these options are extensively documented (currently it is https://docs.arangodb.com/devel/Manual/Administration/Configuration/RocksDB.html). This way we do not duplicate text.  We are trying to improve how options are handled in the manual, and aggregate things (adding a reference link). If these options are not documented well in https://docs.arangodb.com/devel/Manual/Administration/Configuration/RocksDB.html, we can update that page and add extra next text

Reply:
this page can be used as well instead of the previous - this is the new page about arangodb options: https://docs.arangodb.com/devel/Manual/Programs/Arangod/Options.html#rocksdb-options